cmd/podman/containers/create.go

```diff
@@ -100,3 +106,27 @@ func createInit(c *cobra.Command) error {
 
 	return nil
 }
+
+func pullImage(imageName string) error {
+	br, err := registry.ImageEngine().Exists(registry.GetContext(), imageName)
+	if err != nil {
+		return err
+	}
+	pullPolicy, err := config.ValidatePullPolicy(cliVals.Pull)
+	if err != nil {
+		return err
+	}
+	if !br.Value || pullPolicy == config.PullImageAlways {
+		if pullPolicy == config.PullImageNever {
+			return errors.New("unable to find a name and tag match for busybox in repotags: no such image")
+		}
+		_, pullErr := registry.ImageEngine().Pull(registry.GetContext(), imageName, entities.ImagePullOptions{
+			Authfile: cliVals.Authfile,
+			Quiet:    cliVals.Quiet,
+		})
+		if pullErr != nil {
+			return pullErr
+		}
+	}
+	return nil
+}
```

Comment:
This looks like it pulls the image even if `--pull` is not specified?  Is that correct behavior?

Also the man page fo create only lists `--pull` and not `--pull-always` or `--pull-never` should that change?

Reply:
docker behavior, fwiw, is to NOT pull on create and require it to be local.

Reply:
podman v1.9 pulls automatically.

pull-always and --pull-never only exists for buildah.  Not for create.